### PR TITLE
Fix CI for arduino-cli 0.28

### PR DESCRIPTION
**Pull request by Acuadros95**
Signed-off-by: acuadros95 <acuadros1995@gmail.com>

### CODE REVIEW

**Review by pablogs9 — COMMENTED** [review]
Can we remove those lines?

**Comment by Acuadros95** [comment]
@mergify backport foxy galactic main

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ jobs:
               additional_urls:
                 - https://raw.githubusercontent.com/ROBOTIS-GIT/OpenCR/master/arduino/opencr_release/package_opencr_index.json
                 - https://github.com/espressif/arduino-esp32/releases/download/2.0.2/package_esp32_index.json''' > arduino-cli.yaml
+            curl -fsSL https://raw.githubusercontent.com/ROBOTIS-GIT/OpenCR/master/arduino/opencr_release/package_opencr_index.json -o /github/home/.arduino15/package_opencr_index.json
```
Comment (pablogs9) [inline .github/workflows/ci.yml:31]: Should we remove the above lines?

**Review by pablogs9 — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Acuadros95) [reply .github/workflows/ci.yml:31]: No, they still need to be added to the additional repositories.

**Pull request merged**